usr/share/doc/find-the-command/ftc.zsh

```diff
@@ -1,13 +1,19 @@
 # Print to stderr
 alias _cnf_print='echo -e 1>&2'
 
+_cnf_actions=('install' 'info' 'list files' 'list files (paged)')
+
 # Parse options
 for opt in $*
 do
     case $opt in
         noprompt) cnf_noprompt=1 ;;
         su) cnf_force_su=1 ;;
         quiet) cnf_verbose=0 ;;
+        install) cnf_action=$_cnf_actions[1] ;;
```

Comment:
I don't really understand the point of these

This case switch is for some basic configuration via options appended to the `source *.zsh` command

Reply:
This adds the option `install` which skips [the action selection](https://github.com/agur4ik/find-the-command/pull/3/files#diff-8b576982a090b417663c3a0676b58f2fR82) and jumps straight to package installation just like if the user selected install.

Reply:
Okay, now I get it.